src/components/panels/SpoolmanPanel.vue

```diff
@@ -2,7 +2,13 @@
     <div>
         <panel :icon="mdiAdjust" :title="title" card-class="spoolman-panel" :collapsible="true">
             <template #buttons>
-                <v-btn icon tile :title="changeSpoolTooltip" @click="showChangeSpoolDialog = true">
+                <spoolman-tools-dropdown :tools="toolsWithSpoolId" />
```

Comment:
hide new dropdown when no tool macros are present or they don't contain the `variable_spool_id` field.

```suggestion
                <spoolman-tools-dropdown v-if="toolsWithSpoolId.length !== 0" :tools="toolsWithSpoolId" />
```

Reply:
I pushed a fix for that, but i did it a little bit different, because the next line also has a if condition.